### PR TITLE
Document memory and CPU requests and limits

### PR DESCRIPTION
**Pull request by jean-edouard**

### CODE REVIEW

File: docs/virtual_machines/resources_requests_and_limits.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+metadata:
+  labels:
+    autoCpuLimit: "true"
+    kubernetes.io/metadata.name: default
+  name: default
+```
+
+## Memory
+### Memory requests on the container
+- VM(I)s must specify a desired amount of memory, in either spec.domain.memory.guest or spec.domain.resources.requests.memory (ignoring hugepages, see the [dedicated page](../operations/hugepages.md)). If both are set, the memory requests take precedence. The container will request that amount + a calculated overhead.
```
Comment (dhiller) [inline docs/virtual_machines/resources_requests_and_limits.md:51]: I'd suggest replacing the `+ ....` with `, adding a calculated overhead` or `, including an added calculated overhead`

File: docs/virtual_machines/resources_requests_and_limits.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### Memory limits on the container
+- By default, no memory limit is set on the container
+- Manually setting a memory limit on the VM(I) will set the same value on the container
+
+#### Warnings
+- Memory limits have to be more than memory requests + overhead
```
Comment (dhiller) [inline docs/virtual_machines/resources_requests_and_limits.md:58]: Maybe we should describe what happens in that case.

**Review by dhiller — COMMENTED** [review]
/approve

Thank you @jean-edouard !

Two nits inline.

**Comment by jean-edouard** [comment]
@dhiller thank you for the review! I tried to address your comments, English is hard... PTAL!

**Comment by vladikr** [comment]
Looks great!

**Review by dhiller — APPROVED** [review]
Thank you, this version is even better!

**Pull request merged**